test/unit/cmap/message_stream.test.js

```diff
@@ -19,6 +19,34 @@ function bufferToStream(buffer) {
 }
 
 describe('Message Stream', function () {
+  context('when the stream uses the streaming protocol', function () {
```

Comment:
could be good to add an analogous test to show that all messages are read outside the streaming protocol